dev_docs/01_getting_started.md

```diff
@@ -4,8 +4,8 @@ The design system has two primary parts, the UI component library [`/lib`](../li
 
 ## Prerequisites
 
-- Node.js 10.x (see [Node Version Manager](https://github.com/nvm-sh/nvm))
-- Yarn 1.x
+- Node.js 18.x (see [Node Version Manager](https://github.com/nvm-sh/nvm))
+- Yarn >=1.22.22
```

Comment:
Now when I run `yarn lint`, `yarn dev` I get this error because I dont have pnpm installed:

```
> yarn lint
yarn run v1.22.22
$ kolibri-tools lint --pattern '**/*.{js,vue,scss,less,css}' --ignore '**/node_modules/**,**/.nuxt/**,**/dist/**,**/lib/KIcon/precompiled-icons/**,**/lib/keen/**,**/docs/environment.js,**/docs/jsdocs.js'

Kolibri CLI: Error: Command failed: "pnpm --version"

stderr:
Volta error: Could not find executable "pnpm"

Use `volta install` to add a package to your toolchain (see `volta help install` for more info).
Error details written to /home/alexvelezll/.volta/log/volta-error-2024-05-27_11_10_15.764.log


original error message:
Command failed: pnpm --version
Volta error: Could not find executable "pnpm"

Use `volta install` to add a package to your toolchain (see `volta help install` for more info).
Error details written to /home/alexvelezll/.volta/log/volta-error-2024-05-27_11_10_15.764.log


error Command failed with exit code 1.
```

Does the new version of kolibri-tools require pnpm to be installed? If so I think we need to add this to our prerequisites.

Reply:
Hmm, no, that didn't solve it. It's weird because I tried to run it on my Windows machine, where I also don't have Pnpm, and it worked there :thinking:

In my Ubuntu machine I finally installed pnpm it through `volta install pnpm` and then `yarn lint` worked.